### PR TITLE
[#201] Fix for exposed port on prod for graphelier news

### PR DESCRIPTION
**Pull request by alvyn279**
Fix for a port issue on prod.

### CODE REVIEW

File: news/service/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ENV FLASK_APP=server.py
 ENV FLASK_DEBUG=1
 ENV FLASK_ENV=development
 EXPOSE 8080
-CMD ["python", "server.py"]
+CMD [ "python", "-m", "flask", "run", "--host", "0.0.0.0", "--port", "8080" ]
```
Comment (Lercerss) [inline news/service/Dockerfile:11]: How will this affect dev/hot-reload?

Comment (alvyn279) [reply news/service/Dockerfile:11]: still works fine :)

Comment (Lercerss) [reply news/service/Dockerfile:11]: Interesting, is `server.py` still used?

Comment (alvyn279) [reply news/service/Dockerfile:11]: yeah through env variable `FLASK_APP=server.py`

**Pull request merged**